upgradeable_contract/new_contract/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+#![no_std]
+
+use soroban_sdk::{contractimpl, contracttype, Address, BytesN, Env};
+
+#[contracttype]
+#[derive(Clone)]
+enum DataKey {
+    Admin,
+}
+
+pub struct UpgradeableContract;
+
+#[contractimpl]
+impl UpgradeableContract {
```

Comment:
nit: I think it might be useful to add some simple new function here to demonstrate that the interface can be extended this way as well.

Reply:
Done